### PR TITLE
Add More Dependencies

### PR DESCRIPTION
**Pull request by derekirish3**
I found more dependencies required to link llvm. Would be nice to add them here. You can validate this here:

https://github.com/derekirish3/llrl/blob/main/mayhem/Dockerfile
https://github.com/derekirish3/llrl/actions/runs/5097274852/jobs/9163679708

### CODE REVIEW

File: fuzz-rust/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,17 @@
 FROM rustlang/rust:nightly
 
 RUN apt-get update && apt-get upgrade -y && \
-    apt-get install -y build-essential clang-11 llvm-11-tools cmake clang curl binutils-dev libbfd-dev libunwind8-dev libblocksruntime-dev pkg-config libudev-dev libssl-dev && \
+    apt-get install -y build-essential clang-11 llvm-11-tools cmake clang curl binutils-dev libbfd-dev libunwind8-dev libblocksruntime-dev pkg-config libudev-dev libssl-dev libgc-dev libtool-bin autoconf && \
```
Comment (nottirb) [inline fuzz-rust/Dockerfile:4]: This looks good, more useful dep's is fine with me.

File: fuzz-rust/Dockerfile [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
     ln -s /usr/bin/llvm-config-11 /usr/bin/llvm-config
 
+RUN wget http://www.hboehm.info/gc/gc_source/gc-8.0.4.tar.gz \
+	&& tar xzf gc-8.0.4.tar.gz \
+	&& cd gc-8.0.4 \
+	&& ./configure \
+	&& make \
+	&& make install
+
```
Comment (nottirb) [inline fuzz-rust/Dockerfile]: 2 things:
- Spacing is tabs, but the rest of the file is spaces, please fix to match
- Isn't gc covered by libgc-dev, a dependency you added?

**Review by nottirb — CHANGES_REQUESTED** [review]
I verified that your build compiles, can you verify that the gc library you installed with wget isn't covered by libgc-dev? I think it should be?

File: fuzz-rust/Dockerfile (hunk 2 above)
Comment (derekirish3) [reply fuzz-rust/Dockerfile]: Looking into this now, I'll get back to you ASAP. Thanks for the fast response.

Comment (derekirish3) [reply fuzz-rust/Dockerfile]: Yes everything works as expected without adding the wget. Removing it from the PR, nice catch!

Comment (nottirb) [reply fuzz-rust/Dockerfile]: Thanks for verifying that, I'll make sure everything builds as expected, and then I'll merge and update the package immediately.

**Comment by nottirb** [comment]
LGTM! Thanks!

**Pull request merged**